File: array/1470_shuffle_the_array/1470_shuffle_the_array.go
package main

import "fmt"

func shuffle(nums []int, n int) []int {
	res := []int{}
	for i := 0; i < n; i++ {
		res = append(res, nums[i])
		if i+n < 2*n {
			res = append(res, nums[i+n])
		}
	}
	return res
}

func main() {
	nums := []int{2, 5, 1, 3, 4, 7}
	res := shuffle(nums, 3)
	fmt.Println(res)
}

File: array/1470_shuffle_the_array/1470_shuffle_the_array.py
from typing import List


class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        res = []
        for i, j in zip(nums[0:n],nums[n:]):
            res+=[i,j]
        return res


res = Solution().shuffle([2,5,1,3,4,7], 3)
print(res)